events/hfe_events_features.txt

```diff
@@ -99,47 +385,89 @@ country_event = {
 	id = hfe_features.4
 	hide_window = yes
 	is_triggered_only = yes
+	show_sound = shattered_planet_explosion
 	trigger = {
-		has_valid_civic = hfe_civic_features_exodus
+		has_origin = hfe_civic_features_exodus
 	}
 	immediate = {
 		set_country_flag = custom_start_screen
 		set_country_flag = first_colony
+
 		give_technology = {
-			message = no
+			message = yes
 			tech = hfe_tech_features_exodus
 		}
+
 		add_modifier = {
 			modifier = hfe_discovery_translator
+			days = -1
 		}
+
+		set_country_flag = is_government_in_exile
+
+		give_technology = {
+			tech = tech_assault_armies
+			message = no
+		}
+
 		set_country_type = hfe_features_exodus
-		if = {
-			limit = {
-				is_militarist = yes
-			}
-			create_fleet = {
-				effect = {
-					set_owner = root
-					while = {
-						count = 3
-						create_army_transport = {
-							army_type = assault_army
+
+		every_owned_planet = {
+			if = {
+				limit = {
+					NOT = {
+						owner = { is_pacifist = yes }
+					}
+					has_global_flag = hfe_hardcore_disable
+
+				}
+			
+				owner = {
+					create_fleet = {
+						name = "Real Estate Agents"
+						settings = {
+							can_upgrade = yes
```

Comment:
Why this line

Reply:
The intent was for a non-Pacifist Exodus nation to spawn with armies. This allows it, possibly, to invade Primitives and found their colony that way.